### DISCUSSION TITLE
Need Help

### DISCUSSION THREAD

**Discussion by sh1neryu** [Chia-Network/chia-blockchain#6441] [Support]
can anyone explain why my wallet and farmer are always 0? with the condition of wallet syncron and farmer running? 
![Screenshot_14](https://user-images.githubusercontent.com/29735918/120877933-5f2d8980-c5e3-11eb-8f8e-0cc7e5c5ba49.png)
![Screenshot_16](https://user-images.githubusercontent.com/29735918/120877940-65bc0100-c5e3-11eb-86b3-8a37ba050df8.png)
thanks before

**Comment by rafi7777** [comment]
I have the same problem!
I have been growing CHIA for two months. I currently have 192 plots on one computer and 72 on the other. I never got anything, no coins, only the difficulty increases every day, faster than I am creating new plots! I'm starting to wonder if this makes any sense!?! Or maybe something is wrong with the software -HELP!

**Comment by BrandtH22** [comment]
@sh1neryu , if you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
